apps/faucet/src/App/Faucet.tsx

```diff
@@ -104,7 +106,7 @@ export const FaucetForm: React.FC<Props> = ({ isTestnetLive }) => {
         transfer: {
           target: sanitizedTarget,
           token: tokenAddress,
-          amount: amount * 1_000_000,
+          amount: (amount || 0) * 1_000_000,
```

Comment:
You can probably just return early from this function if `amount` is undefined since we don't want to submit in that case, the same way `targetAddress` is handled on line 64.

Reply:
Fixed!